src/CMakeLists.txt

```diff
@@ -191,6 +191,11 @@ if(MSVC AND MSVC_VERSION LESS 1800)
   set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /TP")
 endif()
 
+# On MSVC 15.0 and newer, compile with C11 standard
+if (MSVC AND MSVC_VERSION GREATER_EQUAL 1910)
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /std:c11")
+endif()
```

Comment:
Please undo this.  We should be less clever and let CMake do the work for us.  Instead of these changes, at the top level CMakeLists.tx,t, let's do this:

```
if(MSVC)
  set(CMAKE_C_STANDARD 11)
  set(CMAKE_C_STANDARD_REQUIRED YES)
endif()
```

We can un-conditionalize it as well, which might be better.  But that lets CMake figure out the details of the version of the compiler, its support, and the appropriate flags.

Reply:
I don't have a problem with baselining at C99 and requesting C11 for MSVC.  It is merely the mechanism by which we do so.  I would be perfectly happy with:

```cmake
if(MSVC)
  set(CMAKE_C_STANDARD 11)
else()
  set(CMAKE_C_STANDARD 99)
endif()
set(CMAKE_C_STANDARD_REQUIRED YES)
```